modules/@apostrophecms/schema/ui/apos/components/AposArrayEditor.vue

```diff
@@ -236,6 +235,25 @@ export default {
       const currentIndex = this.next.findIndex(item => item._id === this.currentId);
       this.next[currentIndex] = this.currentDoc.data;
     },
+    isModified() {
```

Comment:
I don't see a `return false` case here.

Reply:
undefined is falsy, but clarity is good.